packages/editor/src/components/StructureTree/ComponentTree.tsx

```diff
@@ -3,48 +3,30 @@ import { Box, Text, VStack } from '@chakra-ui/react';
 import { ComponentSchema } from '@sunmao-ui/core';
 import { ComponentItemView } from './ComponentItemView';
 import { DropComponentWrapper } from './DropComponentWrapper';
-import { ChildrenMap } from './StructureTree';
 import { genOperation } from '../../operations';
 import { EditorServices } from '../../types';
 import { observer } from 'mobx-react-lite';
+import { isEqual } from 'lodash-es';
```

Comment:
`lodash-es` will have problems with ssr

Reply:
The SSR PR hasn't been merged into develop branch yet. I will change it later if the SSR PR is merged before this PR.